pointcloud/voxel.go

```diff
@@ -201,7 +201,7 @@ func (v1 *Voxel) CanMerge(v2 *Voxel, angleTh, cosTh float64) bool {
 func (v1 *Voxel) ComputeCenter() {
 	center := r3.Vector{}
 	for _, pt := range v1.Positions() {
-		center.Add(pt)
+		center = center.Add(pt)
```

Comment:
I *believe* this is the right fix, because otherwise this function does nothing and alters no data outside of itself.

Reply:
@bhaney Not sure if this your area or not... I can't see where this function is used, so wasn't sure if the fix works.